openassessment/templates/openassessmentblock/grade/oa_grade_complete.html

```diff
@@ -144,7 +144,7 @@ <h4 id="feedback__overall__prompt__{{ xblock_id }}" class="field-group__label">{
                                                            class="option__input feedback__overall__value"
                                                            value="These assessments were useful."
                                                            aria-labelledby="feedback__overall__prompt__{{ xblock_id }}"/>
-                                                    <label class="option__label" for="feedback__overall__value--useful">{% trans "These assessments were useful." %}</label>
+                                                    <label class="option__label" for="feedback__overall__value--useful__{{ xblock_id }}">{% trans "These assessments were useful." %}</label>
```

Comment:
These checkboxes are part of a survey for learners to say how useful their peer grading experience was.

I just happened to notice that these labels were not correctly linked to their inputs (Pycharm had highlighted them as red).

As far as I know, there is no way to enable this part of the HTML via Studio. I manually edited the HTML file to remove the conditional that hides these options on my devstack, but these won't be enabled on the sandbox.